roles/provision_vcenter/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Validate Inputs
+  ansible.builtin.import_tasks: "{{ role_path }}/tasks/validate_inputs.yml"
+
+- name: Check If VCenter Already Exists
+  community.vmware.vmware_guest_info:
+    hostname: "{{ provision_vcenter_auth_hostname }}"
+    username: "{{ provision_vcenter_auth_username }}"
+    password: "{{ provision_vcenter_auth_password }}"
+    port: "{{ provision_vcenter_auth_port | default(omit) }}"
+    validate_certs: "{{ provision_vcenter_auth_validate_certs | default(omit) }}"
+    datacenter: "{{ provision_vcenter_datacenter_name | default('') }}"
+    name: "{{ provision_vcenter_vm_name }}"
+  register: _vcenter_lookup
+  failed_when: false
+
+- name: Create Instance From ISO
+  ansible.builtin.include_tasks: "{{ role_path }}/tasks/deploy_from_iso.yml"
```

Comment:
This path can be relative, not absolute. So this shouldbe enough:
```
ansible.builtin.include_tasks: "deploy_from_iso.yml"
```

Also in other parts of file. Also I would use `include_tasks` or `import_tasks` for whole role. You use each time different one.

Reply:
ah good catch, thank you